src/main/host/syscall/handler/clone.rs

```diff
@@ -376,8 +376,74 @@ impl SyscallHandler {
         )
     }
 
-    #[log_syscall(/* rv */kernel_pid_t)]
+    #[log_syscall(/* rv */ kernel_pid_t)]
     pub fn gettid(ctx: &mut SyscallContext) -> Result<kernel_pid_t, SyscallError> {
         Ok(kernel_pid_t::from(ctx.objs.thread.id()))
     }
+
+    #[log_syscall(/* rv */ std::ffi::c_int,
+        /* hdrp */ *const std::ffi::c_void,
+        /* datap */ *const std::ffi::c_void)]
+    pub fn capget(
+        ctx: &mut SyscallContext,
+        hdrp: ForeignPtr<linux_raw_sys::general::__user_cap_header_struct>,
+        datap: ForeignPtr<[linux_raw_sys::general::__user_cap_data_struct; 2]>,
```

Comment:
Note that the structs here (and in `capset`) would change to `linux-api` versions (as would the `LINUX_CAPABILITY_VERSION_3` constant below).